src/authentication.ts

```diff
@@ -390,6 +395,7 @@ export function addAuthenticationAuthority(sapi: SakuraApi, options: IAuthentica
             [fields.passwordStrengthDb]: this.getPasswordStrength(newPassword, user)
           });
         })
+        .then(() => options.onChangePasswordEmailRequest(user, req, res))
```

Comment:
This should not fire if it's not defined... that'll generate an error (see comments above about needing to make this optional. Something like

`.then(() => (options.onChangePasswordEmailRequest) ? options.onChangePasswordEmailRequest(user, req, res) : Promise.resolve())`